waku/node/v2/rpc/wakucallsigs.nim

```diff
@@ -6,6 +6,7 @@ proc waku_publish(topic: string, message: seq[byte]): bool
 proc waku_publish2(topic: string, message: seq[byte]): bool
 proc waku_subscribe(topic: string): bool
 proc waku_query(topics: seq[string]): bool
+proc waku_subscribe2(topic: string, contentFilters: seq[seq[string]]): bool
```

Comment:
FWIW the publish2 moniker was just an ugly hack I did as I was fixing stuff from nim-libp2p rebase. It is not a good pattern. I'd call this waku_subscribe_content/filter perhaps?